test/any/web_apis/web_apis-integration.es6.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/*
+ * The object graph files under /test/data are extracted using
+ * web-apis (https://github.com/mdittmer/web-apis). The test
+ * specifications are defined at:
+ * https://docs.google.com/spreadsheets/d/1bnm-atGD2SPyQ5XBltEXOaVhzK0ey5Dvpe4Nwu2m588
+ **/
+
+describe('WebAPI and api extractor', function() {
+  let chrome56 = global.DATA.chrome56;
+  let edge14 = global.DATA.edge14;
+  let safari602 = global.DATA.safari602;
+  let og = global.ObjectGraph;
+  let extractor = org.chromium.apis.web.apiExtractor.create({});
```

Comment:
Loading it every time would definitely be better. Test run time is also important, but I'd rather have slightly longer hermetic tests than faster and potentially flaky (or worse, incorrect) tests.